src/app/content/components/Sidebar.tsx

```diff
@@ -7,17 +8,21 @@ import * as actions from '../actions';
 import { isArchiveTree } from '../guards';
 import * as selectors from '../selectors';
 import { ArchiveTree, Book, Page } from '../types';
-import { stripIdVersion } from '../utils';
+import { scrollTocSectionIntoView, stripIdVersion } from '../utils';
 import ContentLink from './ContentLink';
 
+const sidebarOpenWidth = 300;
+const sidebarClosedWidth = 40;
+const sidebarClosedOffset = sidebarOpenWidth - sidebarClosedWidth;
+
 const SidebarPlaceholder = styled.div<{open: boolean}>`
   background-color: white;
   overflow-x: hidden;
   transition: all 300ms;
-  width: 40px;
+  width: ${sidebarClosedWidth}px;
```

Comment:
Rather than overriding this later, would it be clearer if it was something like:

```suggestion
  ${(props) => !props.open && css`
    width: 300px;
    width: ${sidebarClosedWidth}px;
  `}
```